membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -177,7 +178,7 @@ class TouchpointComponents(
       accessClaimsParser = UserFromTokenParser,
     )
   }
-  lazy val identityAuthService = new IdentityAuthService(identityPlayAuthService)
+  lazy val identityAuthService = new services.IdentityAuthService(identityPlayAuthService)
```

Comment:
I needed to organise imports to get scalafmt to pass, but if I did, it picked up the wrong `IdentityAuthService` so I just fully qualified it